.classpath

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
you should not commit files like this. Please update git ignore file

https://www.w3schools.com/git/git_ignore.asp?remote=github

https://github.com/github/gitignore